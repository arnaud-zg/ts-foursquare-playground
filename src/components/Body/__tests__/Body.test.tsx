import React from 'react'
import renderer from 'react-test-renderer'
import { Body } from '..'

describe('Component/Body', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Body>
          <p>Custom content</p>
        </Body>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
