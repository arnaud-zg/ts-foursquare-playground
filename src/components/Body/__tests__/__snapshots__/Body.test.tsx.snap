// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/Body renders correctly 1`] = `
<body
  className=""
>
  <p>
    Custom content
  </p>
</body>
`;
